core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -405,9 +406,15 @@ public void listStatus() throws Exception {
     Path path = new Path("/dir");
     alluxio.client.file.FileSystem alluxioFs =
         mock(alluxio.client.file.FileSystem.class);
-    when(alluxioFs.listStatus(new AlluxioURI(HadoopUtils.getPathWithoutScheme(path))))
-        .thenReturn(Lists.newArrayList(new URIStatus(fileInfo1), new URIStatus(fileInfo2)));
     FileSystem alluxioHadoopFs = new FileSystem(alluxioFs);
+    URI uri = URI.create(Constants.HEADER + "host:1");
+    alluxioHadoopFs.initialize(uri, getConf());
+    ListStatusPOptions listStatusPOptions = ListStatusPOptions.getDefaultInstance().toBuilder()
+        .setExcludeMountInfo(alluxioHadoopFs.mAlluxioConf.getBoolean(
+            PropertyKey.USER_HDFS_CLIENT_EXCLUDE_MOUNT_INFO_ON_LIST_STATUS)).build();
```

Comment:
In your UT this is actually testing with `USER_HDFS_CLIENT_EXCLUDE_MOUNT_INFO_ON_LIST_STATUS=false` right? I think we need a bit coverage with `USER_HDFS_CLIENT_EXCLUDE_MOUNT_INFO_ON_LIST_STATUS=true`. So one simple thing you can do is to duplicate these tests and manually give them 
```
// in one test
.setExcludeMountInfo(alluxioHadoopFs.mAlluxioConf.getBoolean(true).build();
// And in the other test
.setExcludeMountInfo(alluxioHadoopFs.mAlluxioConf.getBoolean(false).build();
```

It's even better if you can add 1 or 2 more UT, specially testing your logic (where the FileInfo does not have such fields).

Reply:
I have add another test to test true case.
But testing my logic (where the FileInfo does not have such fields) is a little diffcult.